sample_size/multiple_testing.py

```diff
@@ -0,0 +1,64 @@
+from typing import List
+import numpy as np
+from statsmodels.stats.multitest import multipletests
+
+from sample_size.metrics import BaseMetric
+
+
+class MultipleTestingMixin:
+    """
+    This class calculates sample size required under the case of multiple testing
+
+    Attributes:
+    metrics: a list of BaseMetric registered by users
+    variants: number of variants, including control
+    alpha: statistical significance
+    power: statistical power
+
+    """
+    metrics: List[BaseMetric]
+    alpha: float
+    power: float
+    variants: int
+
+    def get_multiple_sample_size(self) -> int:
+        if len(self.metrics) < 2:
+            return self.get_single_sample_size(self.metrics[0])
+        lower = max([self.get_single_sample_size(metric) for metric in self.metrics])
```

Comment:
`self.get_single_sample_size` does not exist in `MultipleTestingMixin`, I don't think we should call it here

Reply:
> can you rebase your branch after you merge #30 so the earlier updates are not showing in this PR

Actually I did pull most changes from #30, otherwise it's blocking this PR. I will sync the last few changes after its merged